_search-plugins/search-relevance/ml-inference-rerank-by-field.md

```diff
@@ -0,0 +1,277 @@
+---
+layout: default
+title: ML Inference Processor with By Field Rerank type
+parent: Reranking search results
+grand_parent: Search relevance
+has_children: false
+nav_order: 20
+---
+# ML Inference Processor with By Field Rerank type
+Introduced 2.18
+{: .label .label-purple }
+
+You can use the results of a remote model via the [ml_inference]({{site.url}}{{site.baseurl}}/_ingest-pipelines/processors/ml-inference.md) processor, with a [by_field]({{site.url}}{{site.baseurl}}/search-plugins/search-relevance/rerank-by-field/) rerank type to get better search results.
```

Comment:
@mingshl Do we support local model for rerank?

Reply:
if a local model with multiple tensors in the model output, it's not supported yet. @rbhavna is working on a RFC to solve this limitation.

Reply:
Does local cross encoder model return multiple tensors ?

Reply:
yes. the local text_similarity type of model returns multiple tensors